libs/core/algorithms/include/hpx/parallel/unseq.hpp

```diff
@@ -0,0 +1,9 @@
+//  Copyright (c) 2022 A Kishore Kumar
+//
+//  SPDX-License-Identifier: BSL-1.0
+//  Distributed under the Boost Software License, Version 1.0. (See accompanying
+//  file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+
+#pragma once
+
+#include <hpx/config.hpp>
```

Comment:
Why do you need this file? Wouldn't #including <hpx/config.hpp> instead of this file be sufficient?